site/content/entry/citra-progress-report-2023-q1/index.md

```diff
@@ -0,0 +1,390 @@
++++
+date = "2023-02-10T18:15:00+00:00"
+title = "Citra Mega Progress Report 2020 Q2~2023 Q1"
+tags = [ "progress-report" ]
+author = "autumnburra"
+coauthor = "flTobi"
+coauthor2 = "sleepingsnake"
++++
+
+Welcome back to another Citra Mega Progress Report!
+
+Yes! We aren't dead! First off, we do apologize for the wait, we are still lacking a full time writer. 
+In the meantime, Citra community moderator [autumnburra](https://community.citra-emu.org/u/autumnburra/summary) and developer [FearlessTobi](https://community.citra-emu.org/u/fltobi/summary) have come together with the assistance of another community moderator, [SleepingSnake](https://community.citra-emu.org/u/sleepingsnake/summary), to provide you, the Citra community, with this awesome report on all the changes we’ve had in Citra since 2020 Q2!
+
+If you keep up to date with messages posted in the `#development` channel in our [Discord server](https://citra-emu.org/discord/), you may already be aware of some of these! But nevertheless, this is a read that you’re not gonna want to miss out on! 
```

Comment:
Missed this earlier, "If you keep up to date with..." should be in the Present Perfect Continuous tense. So "If you('ve)/have been keeping up to date with..." (since an "action" started in the past and is continuing into the present).